cache-hazelcast/src/main/java/io/micronaut/cache/hazelcast/HazelcastMemberConfiguration.java

```diff
@@ -25,15 +25,42 @@
  *
  * @since 1.0.0
  */
-@ConfigurationProperties(value = "hazelcast", includes = {"properties", "licenseKey", "instanceName", "clusterName"})
+@ConfigurationProperties(value = "hazelcast", includes = {"properties", "licenseKey", "instanceName", "clusterName", "config"})
 @Requires(condition = HazelcastConfigResourceCondition.class)
 @Requires(missingBeans = Config.class)
 @Requires(missingProperty = "hazelcast.client")
 public class HazelcastMemberConfiguration extends Config {
+
+    String config;
+
     /**
      * Default constructor.
      */
     public HazelcastMemberConfiguration() {
         super();
     }
+
+    /**
+     * Returns the path to a Hazelcast XML or YAML configuration file.
+     * <p>If non-null, the contents of the file will override this configuration.
+     * This path will be used to set system property {@code hazelcast.config}.</p>
+     *
+     * @since 4.1
+     * @return The path to the Hazelcast XML or YAML configuration file.
+     */
+    public String getConfig() {
```

Comment:
Add `@Nullable` annotation to return type.

Reply:
resolved via https://github.com/micronaut-projects/micronaut-cache/pull/671/commits/585f0419f6aaab403462fd738628a6dd6acc807f